tribomobile/components/TextInputs.js

```diff
@@ -0,0 +1,83 @@
+import React, {useState} from 'react';
+import {StyleSheet, TextInput, TouchableOpacity} from 'react-native';
+import Colors from '../src/Colors';
+
+const TextInputs = (props) => {
+  const {textInputType, placeholderText} = props;
+
+  const textInputStyling = (textInputType) => {
+    let styleProperties = [];
+    switch (textInputType) {
+      case 'searchBar':
+        styleProperties = [styles.container, styles.searchBar];
+        break;
+      case 'textInputNull':
+        styleProperties = [
+          styles.container,
+          styles.inputTextForm,
+          styles.textInputError,
+        ];
+        break;
+      case 'textInputDisabled':
+        styleProperties = [
+          styles.container,
+          styles.inputTextForm,
+          styles.textInputDisabled,
+        ];
+        break;
+      default:
+        styleProperties = [styles.container, styles.inputTextForm];
+        break;
+    }
+
+    return styleProperties;
+  };
+
+  return (
+    <>
```

Comment:
due you only have one component to return you don't need this tag

Reply:
yep, you can return without using fragments <></>